setup.sh

```diff
@@ -105,6 +105,12 @@ echo "Setting up TnP tools..."
 #git cms-addpkg DataFormats/RecoCandidate
 #git cms-addpkg PhysiscsTools/TagAndProbe
 git cms-merge-topic -u matteosan1:egm_tnp_76X_v1
+git cms-merge-topic -u matteosan1:smearer_76X_test
+git remote add cmssw-ferriff https://github.com/ferriff/cmssw.git
+git fetch cmssw-ferriff
```

Comment:
Could this cherry picks be put into a CMSSW topic branch?

As far as I understand cherry-picks give troubles at remote sites.